pkg/scheduler/apis/config/types.go

```diff
@@ -115,6 +116,16 @@ type KubeSchedulerProfile struct {
 	// is scheduled with this profile.
 	SchedulerName string
 
+	// PercentageOfNodesToScore is the percentage of all nodes that once found feasible
+	// for running a pod, the scheduler stops its search for more feasible nodes in
+	// the cluster. This helps improve scheduler's performance. Scheduler always tries to find
+	// at least "minFeasibleNodesToFind" feasible nodes no matter what the value of this flag is.
+	// Example: if the cluster size is 500 nodes and the value of this flag is 30,
+	// then scheduler stops finding further feasible nodes once it finds 150 feasible ones.
+	// When the value is 0, default percentage (5%--50% based on the size of the cluster) of the
+	// nodes will be scored.
+	PercentageOfNodesToScore int32
```

Comment:
This needs to be added to `staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go` as well

Reply:
I think we can do this after we added the `v1beta2`  API.  
Ref: https://github.com/kubernetes/kubernetes/issues/95446

Reply:
You can do it in both.